src/server.ts

```diff
@@ -113,13 +113,19 @@ const server = () => {
         }
         return (await import(path)).default;
       };
-      const pathname = url.pathname.endsWith("/")
-        ? url.pathname.slice(0, -1)
-        : url.pathname;
       try {
-        return await (await importAPIRoute(pathname))(request);
-      } catch (_error) {
-        return new Response(`Not found`, { status: 404 });
+        const pathname = ensureNoTrailingSlash(url.pathname);
+        const handler = await importAPIRoute(pathname);
+        const response = await handler(request);
+        return response;
+      } catch (error) {
+        console.error(error);
+        return new Response(`Internal Server Error`, {
```

Comment:
Can just be string instead of template literal.

Reply:
Done.